test/unit/express-tooling/authentication-middleware.ts

```diff
@@ -39,7 +39,9 @@ describe('express tooling', () => {
       };
 
       // then
-      expect(() => { authenticationMiddleware(config); }).to.throw(TypeError);
+      expect(() => {
```

Comment:
I am not sure if this is more readable.
I suggest to leave the arrow function on one line in this case. Maybe just leave it as it was.

```
expect(
  () => { authenticationMiddleware(config); }
).to.throw(TypeError);
```

Reply:
how about dropping the curly braces?